react/components/molecules/snackbar-ios/snackbar-ios.js

```diff
@@ -0,0 +1,125 @@
+import React, { PureComponent } from "react";
+import { Animated, Platform, StyleSheet, Text, ViewPropTypes } from "react-native";
+import PropTypes from "prop-types";
+import { mix } from "yonius";
+
+import { baseStyles, IdentifiableMixin } from "../../../util";
+
+import { Touchable } from "../../atoms";
+
+export class SnackbarIos extends mix(PureComponent).with(IdentifiableMixin) {

```

Comment:
Didn't we decide that this would be used for both iOS and Android because we can't easily use the Android one? Not sure about this one, but it's how I recall it.

```suggestion
export class Snackbar extends mix(PureComponent).with(IdentifiableMixin) {
```

Reply:
Following my [comment](https://github.com/ripe-tech/ripe-design/issues/317#issuecomment-849804425) proposing making two components for each platform, I tought you [meant](https://github.com/ripe-tech/ripe-design/issues/317#issuecomment-849804425) the same with "...I believe we should use our own implementation both for iOS and Android.", now re-reading it I can see what you meant, was "misleaded" by my inicial comment.